client/src/App.js

```diff
@@ -4,11 +4,27 @@ import Login from './components/login/Login';
 import Register from './components/register/Register';
 import axios from 'axios';
 
+import AppContainer from './components/main/AppContainer';
+
 const GlobalStyles = createGlobalStyle`
-body{
-  font-family: 'Arimo', sans-serif;
-}
+  html {
+    box-sizing: border-box;
+    font-size: 10px;
```

Comment:
Maybe put the font-size inside the p{} instead of HTML? Because it affected the header element's font size if you put inside the HTML.